spec/index.bs

```diff
@@ -537,13 +540,44 @@ By introducing a [=modelVersion=] field we allow the [=credential schema=] to be
 
 These are just a few examples that illustrate the flexibility of the proposed model. It can be extended to support a wide variety of use-cases and make the burden on issuance and verification simpler by facilitating the development of higher-level tooling.
 
+# Examples # {#examples}
+
+## Verifiable Credentials ## {#vc_example}
+
+The following example shows a reference to a [=Credential Schema=] with the identifier <b></b> instide of a Verifiable Credential following the [[VC_DATA_MODEL]]. The example is adopted from <a href="https://w3c.github.io/vc-data-model/#example-18-usage-of-the-credentialschema-property-to-perform-json-schema-validation">Example 18</a> in the specification.
+
+<pre class="example" highlight="json">
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/examples/v1"
+  ],
+  "id": "http://example.edu/credentials/1872",
+  "type": ["VerifiableCredential", "ExampleCredential"],
+  "issuer": "https://example.edu/issuers/565049",
+  "issuanceDate": "2010-01-01T19:73:24Z",
+  "credentialSchema": {
+    "id": "did:example:c276e12ec21ebfeb1f712ebc6f1;id=83644e73-f540-443c-a440-4d0252dc4f1c;version=1.0",
+    "type": "ExampleSchemaValidator"
+  },
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "exampleKey": "exampleValue"
```

Comment:
I think we need to include the schema and credential in this example, and try to provide a more specific example like emailAddress or ideally something more reflective of the workday use case.

Reply:
updated - is that what you intended?

Reply:
yep, this is better.